File: Algorithm1.java

public class Algorithm1 {


	public Algorithm1(){
		int[] userArray; 
	}
	
	 public Algorithm1(int[] a){
		 int[] userArray = a;
	}
	
	public long addElements(int[] userArray){
		long start = System.nanoTime();
		int total = 0;
		for(int i = 0; i <= userArray.length-1; i++ ) {
			 total += userArray[i];
		}
		long end = System.nanoTime();
		long time = start - end
		System.out.println("\n Time of the algorithm: " + time + " Nanoseconds");
		return total, ;
	}



	
	
	
	
	
	
	
	
	
	
	
	
}

File: Algorithm1.py
# -*- coding: utf-8 -*-
"""
Spyder Editor

This is a temporary script file.
"""

import numpy as np  
import time


#addMembers(array) will add all of the elements of an array if the array has ints as members
def addMembers(array):
    begin = time.perf_counter_ns() #determine the start time
    total = 0; #used for holding the addition of members
    for i in array: #goes through the whole array
        total += i
    end = time.perf_counter_ns() #determine the start time
    endtime = end - begin
    print(endtime)
    return total
    
def generateArray():
    n = 50
    A = np.random.randint(1,100,size=(n))
    
    return A

array1 = np.array(generateArray())

total = addMembers(array1)
